superset-frontend/src/dashboard/components/nativeFilters/utils.test.ts

```diff
@@ -16,75 +16,28 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import { Behavior, FeatureFlag } from '@superset-ui/core';
-import * as uiCore from '@superset-ui/core';
+import { Behavior } from '@superset-ui/core';
 import { DashboardLayout } from 'src/dashboard/types';
 import { CHART_TYPE } from 'src/dashboard/util/componentTypes';
 import { nativeFilterGate, findTabsWithChartsInScope } from './utils';
 
-let isFeatureEnabledMock: jest.MockInstance<boolean, [feature: FeatureFlag]>;
-
 describe('nativeFilterGate', () => {
-  describe('with all feature flags disabled', () => {
```

Comment:
I removed those tests as they didn't make sense now since we got rid of the featuer flags